src/algebraic_geometry/elliptic_curve/weierstrass.lean

```diff
@@ -51,8 +51,8 @@ splitting field of `R` are precisely the $X$-coordinates of the non-zero 2-torsi
     if its discriminant is non-zero.
  * `weierstrass_curve.coordinate_ring.is_domain`: the coordinate ring of a Weierstrass curve is
     an integral domain.
```

Comment:
We should be able to generalize this to any domain, because R[W] embeds into (Frac R)[W]. But maybe another PR.

Reply:
What do you mean? I think this assumes `R`  is a domain and `is_domain_of_field` assumes `R` is a field.

Reply:
currently it assumes normalized_gcd_monoid

Reply:
Right, probably the easiest way is to construct the embedding explicitly and use `function.injective.is_domain`, but that's just a bit long and we'll do it in another PR.